_inc/lib/core-api/wpcom-endpoints/object-id-range.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Automattic\Jetpack\Sync\Modules;
+
+class WPCOM_REST_API_V2_Endpoint_Object_Id_Range {
+	public function __construct() {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	public function register_routes() {
+		register_rest_route( 'wpcom/v2', '/object-id-range', array(
+			array(
+				'methods'     => WP_REST_Server::READABLE,
+				'callback'    => array( $this, 'get_range' ),
+				'description' => 'An endpoint to retrieve the minimum and maximum ID from the database table corresponding to the given object type.',
+				'args'        => array(
+					'module_name' => array(
+						'type'    => 'string',
+						'default' => 'posts',
+						'enum'    => array(
+							'posts',
+							'comments',
+							'users',
+							'terms',
+							'term_relationships',
```

Comment:
Hm, since term relationships aren't really a classic object type, I'm a bit confused if they should be allowed. After all, a relationship can't be identified by its ID only, so how would the range work for those?

Reply:
It works in that it will give you x amount of distinct object ids from the term_relationships table. This may correspond to many more rows in the database, but at least it is a limit of sorts.

Reply:
Right, well maybe we could document that somewhere, ideally where we document the batch size parameter.